proto/babylon/btcstkconsumer/v1/tx.proto

```diff
@@ -16,7 +16,7 @@ service Msg {
   // UpdateParams defines a (governance) operation for updating the module
   // parameters. The authority defaults to the x/gov module account.
   rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
-  // RegisterConsumer registers a CZ consumer to the btcstkconsumer module.
+  // RegisterConsumer registers a BSN consumer to the btcstkconsumer module.
   rpc RegisterConsumer(MsgRegisterConsumer) returns (MsgRegisterConsumerResponse);
```

Comment:
```suggestion
  // RegisterConsumer registers a consumer to the btcstkconsumer module.
  rpc RegisterConsumer(MsgRegisterConsumer) returns (MsgRegisterConsumerResponse);
```

how about simply say consumer rather than BSN? BSN is more like a marketing term imo and might raise confusion

same comment on other BSN occurences

Reply:
Sure! I was tempted to do that, but not sure it'll be accepted.